nautobot/circuits/views.py

```diff
@@ -102,6 +103,27 @@ class ProviderUIViewSet(NautobotUIViewSet):
     queryset = Provider.objects.all()
     serializer_class = serializers.ProviderSerializer
     table_class = tables.ProviderTable
+    object_detail_content = ObjectDetailContent(
+        panels=(
+            ObjectFieldsPanel(
+                section=SectionChoices.LEFT_HALF,
+                weight=100,
+                fields="__all__",
+                exclude_fields=["name", "comments"],
+            ),
+            ObjectFieldsPanel(
+                label="Comments",
+                weight=200,
+                section=SectionChoices.RIGHT_HALF,
+                fields=["comments"],
+                value_transforms={"comments": [render_markdown, placeholder]},
+            ),
+            ObjectsTablePanel(
+                weight=100,
```

Comment:
- same `weight` as the `ObjectFieldsPanel` above?
- might be good for clarity to explicitly specify the `section`?